File: WPFApp/Models/Azure/AzureTranslateResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GetDescriptionImageApp.Models.Azure
{
   public class AzureTranslateResponse
    {
        public AzureTranslateDetectedLanguage detectedLanguage { get; set; }

        public List<AzureTranslateTranslation> translations { get; set; }
    }
}

File: WriteDescription/src/app/interfaces/azure-translate-response.ts
import { AzureTranslateDetectedLanguage } from './azure-translate-detected-language';
import { AzureTranslateTranslation } from './azure-translate-translation';

export interface AzureTranslateResponse {
    detectedLanguage: AzureTranslateDetectedLanguage;
    translations: Array<AzureTranslateTranslation>;
}
